controllers/sts.go

```diff
@@ -228,14 +232,18 @@ func createDSStatefulSet(ds *directoryv1alpha1.DirectoryService, sts *apps.State
 										},
 									},
 								},
-								{
```

Comment:
@wstrange do you DS_ADVERTISED_LISTEN_ADDRESS configured here for anything? This is handled in the docker-entrypoint and is also configured differently for MCS.  We could add it as an option in the CRD?

Reply:
Not sure? If we dont need it - lets remove !

Reply:
- [ ] Add back ENV?

Reply:
It breaks any sort of multi cluster deployment.  The DS_ADVERTISED_LISTEN_ADDRESS is altered in the docker-entrypoint.sh(from using`hostname -f`) to ensure it stays unique in a multi-cluster deployment.  Having this hardcoded completely breaks multi-cluster.

Reply:
can we feature gate it?  In other words,
```
if mcsEnable == true then remove/don't set DS_ADVERTISED_LISTEN_ADDRESS
```